maven-plugin/src/main/java/org/openmrs/maven/plugins/packager/config/AbstractPackagerConfigMojo.java

```diff
@@ -110,8 +115,8 @@ public File getCompiledConstantsFile() {
 	 */
 	public Properties loadPropertiesFromFile(File file) throws MojoExecutionException {
 		Properties p = new ConstantProperties();
-		if (getCompiledConstantsFile().exists()) {
-			try (FileInputStream in = new FileInputStream(getCompiledConstantsFile())) {
+		if (file.exists()) {
+			try (FileInputStream in = new FileInputStream(file)) {
 				p.load(in);
```

Comment:
I noticed this issue - appears to be a bug that never caused issues as this method was only ever being called with getCompiledConstantsFile() as the input parameter.  I've fixed it.